src/_guides/openapi/specification/v3.1/understanding-structure/basic-structure.md

```diff
@@ -0,0 +1,317 @@
+
+These components can be referenced throughout the API specification to avoid duplication.
+
+```yaml
+components:
+  schemas:
+    Trip:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: Unique identifier for the trip
+        origin:
+          type: string
+          description: The starting station of the trip
+        destination:
+          type: string
+          description: The destination station of the trip
+        departure_time:
+          type: string
+          format: date-time
+          description: The date and time when the trip departs
+        arrival_time:
+          type: string
+          format: date-time
+          description: The date and time when the trip arrives
+```
+
+Components can also contain reusable parameters, request bodies, responses, and security schemes. 
+
+```yaml
+components:
+
+  requestBodies:
+    TripRequest:
+      description: A request body for creating a new trip.
+      required: true
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Trip'
+
+  securitySchemes:
+    ApiKeyAuth:
+      type: apiKey
+      in: header
+      name: X-API-KEY
+
+  parameters:
+    pageParam:
+      in: query
+      name: page
+      required: false
+      schema:
+        type: integer
+        default: 1
+        description: The page number for pagination.
+```
+
+Understanding `$ref` is a large topic, and we'll get into it later, but just understand for now that this is where reusable components of all sorts are stored.
+
+<!-- TODO Link advanced / splitting documents into multiple parts -->
+
+Another important type of component is the `securitySchemes` section. OpenAPI supports several authentication types, but here are a few examples:
+
+```yaml
+components:
+  securitySchemes:
+    ApiKeyHeader:
+      type: apiKey
+      in: header
+      name: X-API-Key
+    
+    JWT:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+   
+    OAuth2ReadWrite:
+      type: oauth2
+      flows:
+        authorizationCode:
+          scopes:
+            read: Grants read access
+            write: Grants write access
+          authorizationUrl: https://example.com/oauth/authorize
+          tokenUrl: https://example.com/oauth/token
+          refreshUrl: https://example.com/oauth/refresh
+```
+
+This is just a few examples of various types of security schemes that can be defined, but defining them alone doesn't do anything. They need to be referenced by the `security` object.
+
+### 6. Security Object
+
+The top-level `security` object specifies the security schemes that apply globally to the API, so if an entire API uses an API key or OAuth2 you might have:
```

Comment:
```suggestion
The top-level `security` list specifies the security schemes that apply globally to the API, so if an entire API uses an API key or OAuth2 you might have:
```

Reply:
Should we also mention that this `security` list can “override” at the operation level (inside `paths`)?